deploy-path/dist/index.js

```diff
@@ -9497,14 +9497,18 @@ function getDeployProps(gitRefs) {
         };
     }
     if (branch) {
-        const prefixStripMatch = branch.match(/^#?[0-9]{8,}-(.+)$/);
-        const suffixStripMatch = branch.match(/^(.+)-#?[0-9]{8,}$/);
+        const jiraPrefixStripMatch = branch.match(/^[A-Z]{2,}-[0-9]+-(.+)$/);
```

Comment:
Should we make the regex match on branch names like "proj-3-new-feature" as well as names like "PROJ-3-new-feature"?

Reply:
After a discussion on Slack, yes we should handle lower case and mixed case jira ids in branch names. I've updated the PR.